pyiron_base/generic/filedata.py

```diff
@@ -83,55 +84,78 @@ def load_file(filename, filetype=None, project=None):
             :class:`pandas.DataFrame`: containing data from file of filetype = '.csv'
 
     """
-    def _load_txt(file):
-        with open(file, encoding='utf8') as f:
-            return f.readlines()
+    def _load_txt(file, is_file):
+        if is_file:
+            with open(file, encoding='utf8') as f:
+                return f.readlines()
+        else:
+            return file.readlines()
```

Comment:
I'm not sure the extra flag is needed, maybe something like

```suggestion
        if instance(file, io.IOBase):
            return file.readlines()
        else:
            with open(file, encoding='utf8') as f:
                return _load_txt(f)
```
is better.

Reply:
Probably we should then go the whole way and use this check instead of generating the `filename_is_string` at all.